src/test/resources/features/ifs/InterestRateChanges.feature

```diff
@@ -0,0 +1,129 @@
+    Then the ifs service wilL return a total debts summary of
+      | combinedDailyAccrual | interestDueCallTotal | unpaidAmountTotal | totalAmountIntTotal | amountOnIntDueTotal |
+      | 35                   | 17049                | 500000            | 517049              | 500000              |
+    And the 1st debt summary will contain
+      | interestDueDailyAccrual | interestDueDebtTotal | unpaidAmountDebt | totalAmountIntDebt | numberChargeableDays | amountOnIntDueDebt |
+      | 35                      | 17049                | 500000           | 517049             | 0                    | 500000             |
+    And the 1st debt summary will have calculation windows
+      | periodFrom | periodTo   | numberOfDays | interestRate | interestDueDailyAccrual | interestDueWindow | amountOnIntDueWindow | unpaidAmountWindow |
+      | 2020-01-01 | 2020-03-29 | 89           | 3.25         | 44                      | 3962              | 503962               | 500000             |
+      | 2020-03-30 | 2020-04-06 | 8            | 2.75         | 37                      | 301               | 500301               | 500000             |
+      | 2020-04-07 | 2021-03-31 | 359          | 2.6          | 35                      | 12786             | 512786               | 500000             |
+
+  #BUG: Test failing due to issue with interest accrued returned
+  Scenario: Interest rate changes from 0% to 8.5%
+    Given a debt item
+      | originalAmount | dateCreated | dateCalculationTo | mainTrans | subTrans | interestBearing |
+      | 500000         | 2000-01-01  | 2001-03-31        | 1525      | 1000     | true            |
+    And the debt item has no payment history
+    When the debt item is sent to the ifs service
+    Then the ifs service wilL return a total debts summary of
+      | combinedDailyAccrual | interestDueCallTotal | unpaidAmountTotal | totalAmountIntTotal | amountOnIntDueTotal |
+      | 0                    | 0                    | 500000            | 500000              | 500000              |
+    And the 1st debt summary will contain
+      | interestDueDailyAccrual | interestDueDebtTotal | unpaidAmountDebt | totalAmountIntDebt | numberChargeableDays | amountOnIntDueDebt |
+      | 0                       | 0                    | 500000           | 500000             | 0                    | 500000             |
+    And the 1st debt summary will have calculation windows
+      | periodFrom | periodTo   | numberOfDays | interestRate | interestDueDailyAccrual | amountOnIntDueWindow | interestDueWindow | unpaidAmountWindow |
+      | 2000-01-01 | 2000-02-05 | 0            | 0.0          | 0                       | 500000               | 0                 | 500000             |
+      | 2000-02-06 | 2001-03-31 | 418          | 8.5          | 13                      | 500904               | 780               | 500780             |
+
+  #Scenario: Interest rate changes from 1% to 0%

```

Comment:
Does this scenario need to be added?

Reply:
Yes, currently there is no way to implement it based on the test data available